client/src/components/Explorer/Explorer.scss

```diff
@@ -1,12 +1,15 @@
-$c-explorer-bg: #4c4e4d;
-$c-explorer-bg-dark: $nav-grey-1;
-$c-explorer-bg-active: rgba(0, 0, 0, 0.425);
-$c-explorer-secondary: #a5a5a5;
 $c-explorer-easing: cubic-bezier(0.075, 0.82, 0.165, 1);
 $menu-footer-height: 50px;
 
 @import 'ExplorerItem';
 
+:root {
+    --c-explorer-bg: #4c4e4d;
+    --c-explorer-bg-dark: var(--nav-grey-1);
+    --c-explorer-bg-active: rgba(0, 0, 0, 0.425);
+    --c-explorer-secondary: #a5a5a5;
+}
```

Comment:
I don’t think this should be defined at the root level, we don’t want any other component to reuse the explorer’s colors.

Reply:
see https://github.com/wagtail/wagtail/pull/6349#discussion_r494494951 re: restrictions coming from the postcss plugin